src/librustc_resolve/macros.rs

```diff
@@ -621,7 +621,7 @@ impl<'a, 'cl> Resolver<'a, 'cl> {
         }
 
         // Go through all the scopes and try to resolve the name.
```

Comment:
I think this comment may need an update?

I think the large comment above this one may also need a slight tweak?

Reply:
No, we are still going through scopes and trying to resolve, just in slightly different order.
I've updated the larger comment above.